src-site/usage/getting-started.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Getting started
+eleventyExcludeFromCollections: true
+---
+
+## Installation
+
+Install the design system in your project as a python package:
+
+```python
+pip install git+https://github.com/DemocracyClub/design-system.git@python-package
```

Comment:
The bit after the `@` here is the tag or branch name - so now I've merged that branch we'll want to use `git+https://github.com/DemocracyClub/design-system.git@[intended version]`

Reply:
@symroe Where is the version taken from? Usually it would be from package.json, but that's very Node-specific, of course.

Reply:
For the Python package, I [take the version from `package.json`](https://github.com/DemocracyClub/design-system/blob/master/setup.py#L7). However, the above command is pulling in a git revision, so the version would be for a git tag, commit or branch name.

Confusingly, the python package will only be upgraded if the `package.json` version is different in the revision it's told to get  so we need to bump the package version and the git tag at the same time. I think this is fairly typical for a sort of github release publishing system